types/views/views.go

```diff
@@ -39,6 +39,10 @@ func ByteSliceOf[T ~[]byte](x T) ByteSlice[T] {
 	return ByteSlice[T]{x}
 }
 
+// MapKey returns a unique key for a slice, based on its address and length.
+// It does not hash the contents of the slice.
```

Comment:
Remove the sentence about "hash the contents" since that's an implicit property of using `SliceMapKey` with a Go map.